core/kazoo_tasks/include/kazoo_tasks.hrl

```diff
@@ -13,7 +13,7 @@
 -define(KZ_TASKS_INPUT_ERROR_MIME, <<"expected_content">>).
 
 -define(KZ_TASKS_WAIT_AFTER_ROW,
-        ?MILLISECONDS_IN_SECOND * kapps_config:get_integer(?KZ_TASKS_DOC_TYPE, <<"wait_seconds_after_row">>, 2)).
+        kapps_config:get_integer(?KZ_TASKS_DOC_TYPE, <<"wait_milliseconds_after_row">>, 500)).
```

Comment:
These values have traditionally been the name with unit as a suffix. I would use "wait_after_row_ms" in this case.